kratos/tests/test_properties.py

```diff
@@ -21,6 +21,22 @@ def test_copy_properties(self):
         self.assertEqual(properties.GetValue(KM.YOUNG_MODULUS), 1.0)
         self.assertEqual(cloned_properties.GetValue(KM.YOUNG_MODULUS), 10.0)
 
+    def test_properties_utilities(self):
+        current_model = KM.Model()
+
+        model_part= current_model.CreateModelPart("Main")
+
+        properties = model_part.CreateNewProperties(1)
+        properties.SetValue(KM.YOUNG_MODULUS, 1.0)
+        self.assertEqual(properties.GetValue(KM.YOUNG_MODULUS), 1.0)
+
+        properties_copy = model_part.CreateNewProperties(2)
```

Comment:
try to add something so that it is not empty (i am a bit afraid of the clear you do)

Reply:
OK

Reply:
Done